public/config.json

```diff
@@ -1,9 +1,9 @@
 {
-  "apiUrl": "",
-  "ontologyApiUrl": "",
-  "clientId": "",
-  "firecloudUrl": "",
+  "apiUrl": "https://consent.dsde-dev.broadinstitute.org/api",
+  "ontologyApiUrl": "https://consent-ontology.dsde-dev.broadinstitute.org/",
+  "clientId": "1087760099392-u18o6sqvb8uljmdltflohmd34h6hik9b.apps.googleusercontent.com",
+  "firecloudUrl": "https://firecloud-orchestration.dsde-dev.broadinstitute.org/",
   "gwasUrl": "",
-  "nihUrl": "",
+  "nihUrl": "http://mock-nih.dev.test.firecloud.org/link-nih-account/index.html",
```

Comment:
Nit: please revert this file. It really shouldn't be in version control ... I need to make that happen, sorry.